pkg/service/checkpointz/checkpointz.go

```diff
@@ -45,6 +47,33 @@ func (h *Handler) V1Status(ctx context.Context, req *StatusRequest) (*StatusResp
 	return response, nil
 }
 
+// Slot returns the beacon slot for checkpointz.
+func (h *Handler) V1BeaconSlots(ctx context.Context, req *BeaconSlotsRequest) (*BeaconSlotsResponse, error) {
+	response := &BeaconSlotsResponse{}
+
+	slot, err := h.provider.GetFinalizedSlot(ctx)
+	if err != nil {
+		return nil, err
+	}
+	response.Slots = []BeaconSlot{}
+	for i, val := uint64(slot), uint64(slot)-32*50; i > val && i >= 0; i -= 32 {
```

Comment:
o should this be `SlotsPerEpoch` instead of hard coded 32?

Reply:
Yup